.github/workflows/basic-tests.yml

```diff
@@ -148,6 +148,8 @@ jobs:
         env:
           HATCH_ENV: "test"
         working-directory: ./clients/python
+      - name: Compile www assets
+        run: breeze compile-www-assets
```

Comment:
TODO: Generic comment: the tests seem to pass all thests on Python 3.9 + but the Pytest tests do not exit - likely some cleanup needs to happen there. For now switching to DEfault Python only but needs to be fixed before merge.